solidjs-tailwind/src/components/PrivacyBadge/PrivacyBadge.classNames.js

```diff
@@ -0,0 +1 @@
+export const badge = 'py-0.5 px-2 text-xs rounded-xl text-gray-600 border border-gray-300 font-medium';
```

Comment:
Instead of using this method of styling, we should use a CSS module just the wayit was implemented in Nextjs.

This pattern was used in Remix because Remix does not support CSS module.

Reply:
I agree I don't love this pattern either I'll update these